docs/SOPs/data_retraction_SOP.md

```diff
@@ -126,7 +162,21 @@ The metadata spreadsheet can be in many different locations. Please make sure yo
    1. Head to the [Wrangler team Google group](https://groups.google.com/a/data.humancellatlas.org/g/wrangler-team) and delete any email thread that may contain the spreadsheet.
    1. Send an email to the <a href="mailto:wrangler-team@data.humancellatlas.org?subject=[URGENT]%20Dataset%20retraction%20DATASET%20SHORTNAME&body=Hello%0A%0AWe%20have%20been%20asked%20to%20retract%20the%20dataset%20%20%22%3Cshortname_of_dataset%3E%22.%20%0A%0APlease%20delete%20your%20own%20copies%20of%20the%20following%20emails%3A%0A%0A-%20%3Ctitle_of_email_thread%3E%0A%28more%20if%20necessary%29%0A%0APlease%20remove%20any%20local%20copies%20you%20may%20have%20of%20spreadsheets%20associated%20with%20%22%3Cshortname_of_dataset%3E%22%0A%0AIf%20you%20were%20involved%20in%20wrangling%20this%20dataset%2C%20please%20ensure%20no%20local%20copies%20of%20the%20data%2Fmetadata%20are%20left%20in%20the%20EC2%20instance%0A%0AMany%20thanks%20for%20your%20cooperation%0A%0ABest%20regards%2C%0A%0A%3CWrangler%20name%3E">wrangler team</a>
    1. Delete your own local copies of the thread in your mail apps.
-1. [Dev input here] Please ask for a dev to delete the **copy of the spreadsheet associated with the submission** in the ingest database.
+1. Please ask for a dev to delete the **copy of the spreadsheet associated with the submission** in the ingest database.
+
+Instructions for dev to follow:
+
+The spreadsheets are saved in the ingest-broker pod under `/data/spreadsheets`.
+Log into the pod as follows and delete the required spreadsheet(s).
+```
+# switch to env (e.g. prod) context
+kubectx ingest-eks-<env>
+# get ingest-broker-pod name
+kubectl get pods | grep broker | awk '{print $1}'
+kubectl exec -it <ingest-broker-pod> -- /bin/sh 
+```
+
+Spreadsheets are backed up. Need to check backup expiration policy. Same as MongoDB backup.
```

Comment:
Do we have this sorted out?

Reply:
The spreadsheets are not being backed up yet similar to mongo db. I think I'll just remove this line.

Reply:
lgtm!